fuzz/fuzz_targets/fuzz_common.rs

```diff
@@ -353,6 +357,21 @@ pub fn compare_result(
     }
 }
 
+/// When we have different outputs, print the diff
+fn print_diff(rust_output: &str, gnu_output: &str) {
+    println!("Diff=");
+    let diff = TextDiff::from_lines(rust_output, gnu_output);
+    for change in diff.iter_all_changes() {
+        let sign = match change.tag() {
+            ChangeTag::Delete => "-",
+            ChangeTag::Insert => "+",
+            ChangeTag::Equal => " ",
+        };
```

Comment:
This seems to be unnecessary because `ChangeTag` implements `Display`.

Reply:
are you sure ? I can't make it work 

```
fn print_diff(rust_output: &str, gnu_output: &str) {
    let diff = TextDiff::from_lines(rust_output, gnu_output);
    println!("Diff=");
    for change in diff.iter_all_changes() {
        print!("{}", change);
    }
    println!();
}

```

Reply:
Yes :)

`ChangeTag` is only about the sign. So you would have to use:
```
for change in diff.iter_all_changes() {
    print!("{}{}", change.tag(), change);
}
```

Reply:
TIL, thanks :)